examples/voting/voting.py

```diff
@@ -234,20 +236,14 @@ def allowed_to_vote(self, signature: Bytes) -> bool:
             self.snapshot_public_key,
         )
 
+    @subroutine
+    def get_vote_from_box(self, index: UInt64) -> UInt64:
+        return btoi(self.tallies.extract_bytes(index, VOTE_COUNT_BYTES))
```

Comment:
Can this be a .decode() rather than btoi and extract_bytes?

Reply:
if we represented `tallies` as a `arc4.DynamicArray[arc4.UInt64]` then we could do something like `return self.tallies[index].decode()` but then we'd have the overhead storing a length header in box storage unnecessarily. We can't use `arc4.StaticArray` as the length is not known at compile time.

Reply:
we could add some new methods to the `Bytes` type such as `my_bytes.extract_uint64(index)`